torch/csrc/autograd/functions/tensor.cpp

```diff
@@ -71,8 +71,8 @@ auto CopySlices::apply(variable_list&& inputs) -> variable_list {
   auto result = at::empty_strided(base.sizes(), base.strides(), grad.options());
   result.copy_(grad);
 
-  auto offset = view.storage_offset() - base.storage_offset();
-  auto grad_slice = result.as_strided(view.sizes(), view.strides(), offset);
+  TORCH_CHECK(view_fn != nullptr, "view_fn is empty.")
```

Comment:
I am not sure that this is a good error message.
But in general, this should never be possible. So this should be an internal assert no?